app-emulation/containerd/files/containerd.service

```diff
@@ -4,7 +4,7 @@ Documentation=http://github.com/docker/containerd
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc
+ExecStart=/usr/bin/containerd --listen unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim /usr/bin/containerd-shim --state-dir /var/run/docker/libcontainerd/containerd --runtime /usr/bin/runc $CONTAINERD_OPTS
```

Comment:
What do you mean by "break them out"?  Give them separate environment variables?  I didn't have a reason to change them.

My immediate use for this is to set the container start timeout, which is 15 seconds by default, which is not enough time when multiple ARM systems are being emulated on the same host in parallel.